Updated code to keep up with the latest version of extension (3.0.3).

kzvdar42 opened the pull request:
Updated the code & wallpapers IDs file to keep up with the changes in the API.

Also upped the python to 3.4, cause the python 2 is deprecated.

kzvdar42 committed:
Updated the code to support the new API. (And upped the python version to 3.4)

kzvdar42 committed:
Update README.md

kzvdar42 commented:
@cgiAlexis, I've seen you've been trying to update the code, so I hope this helps)

cgiAlexis commented:
This is amazing just to read and see how you've implemented it, thank you!

4llan commented:
Thanks, @kzvdar42. Very nice PR.

I was comparing the (1) `https://earthview.withgoogle.com/download/{}.jpg` vs (2) `https://www.gstatic.com/prettyearth/assets/full/{}.jpg` and noticed this:

- All images are visually the same, except for copyright at the bottom
- Same resolution: 1800x1200
- the filesize of (1) is sometimes higher than (2) because they're not optmized. Found this comment in some (2) files: `Optimized by JPEGmini 3.11.4.3 0x37a6e51a`
- About `5477` and `5478`: (1) returns 404, (2) returns a valid wallpaper

4llan commented:
Missing ids:
- [ ] [1527](https://www.gstatic.com/prettyearth/assets/full/1527.jpg)
- [ ] [1926](https://www.gstatic.com/prettyearth/assets/full/1926.jpg)
- [ ] [5477](https://www.gstatic.com/prettyearth/assets/full/5477.jpg)
- [ ] [7007](https://www.gstatic.com/prettyearth/assets/full/7007.jpg)
- [ ] [7014](https://www.gstatic.com/prettyearth/assets/full/7014.jpg)
- [ ] [7022](https://www.gstatic.com/prettyearth/assets/full/7022.jpg)

kzvdar42 commented:
Where did you get these new ids?

Because I just pulled all ids from their official chrome extension code.

4llan commented:
I've fetched https://earthview.withgoogle.com/_api/photos.json and sorted ids. I've updated my comment above. `5477` and `7014` are ids from my old notes.

kzvdar42 committed:
Handling the 404 page and added new IDs.
Thanks to @4llan for help.

kzvdar42 committed:
Made the tqdm package optional.

kzvdar42 committed:
Fixed small error.

kzvdar42 committed:
Fixed small error.

kzvdar42 committed:
Changed log messages.

kzvdar42 committed:
Updated README.

kzvdar42 committed:
Updated possible `Content-Type`s.

kzvdar42 committed:
Updated README.

kzvdar42 committed:
Fixed small error.

kzvdar42 committed:
Removed the dependency on `requests` package.
Now using `urllib` for requests as in initial work.

kzvdar42 commented:
Thanks, @4llan!
Your interest gave me the motivation to improve the code)
I added the missing ids, removed all dependencies, added handling of the 404 pages and updated the README with yours observations.

kzvdar42 committed:
Fixed an error with new version of `tqdm`.
